### ISSUE TITLE
Default on action date fields not working at all

### ISSUE THREAD

**Issue by ragingdave** [laravel/nova-issues#2850]
- Laravel Version: 7.28.1
- Nova Version: 3.9.2
- PHP Version: 7.4.3
- Browser type and version: Chrome

### Description:
Using default on date fields aren't working at all.
<img width="760" alt="Screen Shot 2020-09-09 at 2 56 27 PM" src="https://user-images.githubusercontent.com/1168344/92641428-ba2c8d00-f2ac-11ea-90e9-20bfd7fef53f.png">

As you can see above I used every permutation of trying to set the default on the date field and none are working correctly.
Here is the corresponding code that built that set of action fields:

### Detailed steps to reproduce the issue on a fresh Nova installation:
```
            Date::make('Date String')
                ->default(today()->toDateString())
                ->rules(['required', 'after_or_equal:today', 'before:+1 week']),
            Date::make('Date String Tomorrow')
                ->default(today()->addDay()->toDateString())
                ->rules(['required', 'after_or_equal:today', 'before:+1 week']),
            Date::make('Date FN String')
                ->default(fn () => today()->toDateString())
                ->rules(['required', 'after_or_equal:today', 'before:+1 week']),
            Date::make('Date FN Tomorrow String')
                ->default(fn () => today()->addDay()->toDateString())
                ->rules(['required', 'after_or_equal:today', 'before:+1 week']),

            Date::make('Date Carbon')
                ->default(today('America/New_York'))
                ->rules(['required', 'after_or_equal:today', 'before:+1 week']),
            Date::make('Date Carbon Tomorrow')
                ->default(today('America/New_York')->addDay())
                ->rules(['required', 'after_or_equal:today', 'before:+1 week']),
            Date::make('Date FN Carbon')
                ->default(fn () => today('America/New_York'))
                ->rules(['required', 'after_or_equal:today', 'before:+1 week']),
            Date::make('Date FN Tomorrow Carbon')
                ->default(fn () => today('America/New_York')->addDay())
                ->rules(['required', 'after_or_equal:today', 'before:+1 week']),
```

The only thing that currently works is to use a hacky workaround from a while back of:
```->withMeta(['value' => today()])```

However, since the patch notes of 3.9.0 explicitly says that Action fields now support providing default values, I would think this workaround should no longer be required.

Relates to #941 (PR), #2796 (Issue), and possibly related to #2848.
Also the related doc link which indicates using a value/callback is appropriate in v3: https://nova.laravel.com/docs/3.0/resources/fields.html#default-values

**Comment by williammoreno** [comment]
Same issue here. Theres a console error:  'TypeError: Cannot read property 'value' of undefined' when ever theres an date set manually via the date picker.

**Comment by davidhemphill** [comment]
This was fixed in v3.9.3 published a few minutes ago. 👍
